cmd/cluster/validatepullsecret.go

```diff
@@ -0,0 +1,92 @@
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var BackplaneClusterAdmin = "backplane-cluster-admin"
+
+func newCmdPullSecret(kubeCli client.Client, flags *genericclioptions.ConfigFlags) *cobra.Command {
```

Comment:
Dumb little request, but could `newCmdPullSecret` (and `newCmdBannedUser`) be updated to match the changes to the actual command (eg `newCmdValidatePullSecret`)?

Super minor prio.  Just nagging at me.